bindings/rust/bench/benches/throughput.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use bench::{
+    CipherSuite::{self, *},
```

Comment:
I just have it because I need to iterate through all of the CipherSuites, and this makes it immediately clear that we're using all of the values of the enum. This also helps with concision with the for loop.